src/contracts/PreAMMBatcher.sol

```diff
@@ -110,21 +111,21 @@ contract PreAMMBatcher {
         for (uint256 i = 0; i < sellOrderToken0.length; i++) {
             require(
                 sellOrderToken0[i].sellToken == sellToken,
-                "sellOrderToken0 are not compatible in sellToken"
+                "invalid token0 order sell token"
```

Comment:
I would expect a need to change the unit tests that actually cause these to revert. Better check that they are working as expected. 

Last time I checked it looked like the `await` needed to be moved outside the `expect` in order to get the assertions to work as desired.

Reply:
Yes, I missed the unit tests which match the revert message. It has since been fixed.